aspnetcore/tutorials/signalr-typescript-webpack.md

```diff
@@ -5,15 +5,15 @@ description: This tutorial provides a walkthrough of bundling and building an AS
 monikerRange: ">= aspnetcore-2.1"
 <!-- ms.author: bradyg -->
 ms.author: wpickett
-ms.custom: mvc
-ms.date: 03/15/2022
+ms.custom: mvc, engagement-fy23
+ms.date: 03/17/2023
 uid: tutorials/signalr-typescript-webpack
 ---
 # Tutorial: Get started with ASP.NET Core SignalR using TypeScript and Webpack
 
 By [Sébastien Sougnez](https://twitter.com/ssougnez) and [Scott Addie](https://twitter.com/Scott_Addie)
 
-:::moniker range=">= aspnetcore-6.0"
+:::moniker range=">= aspnetcore-8.0"
```

Comment:
add old versions warning include above this line.

Reply:
@Rick-Anderson, sorry, what is the "old versions warning include"?  What is the name of the include you are referring to?  I'm looking through our library of includes and I am not spotting what you might be referring to.

Reply:
> @Rick-Anderson, sorry, what is the "old versions warning include"? What is the name of the include you are referring to? I'm looking through our library of includes and I am not spotting what you might be referring to.
> 
> Maybe you are referring to include/signalr-in-shared-framework.md?

```
:::moniker range="< aspnetcore-6.0"
[!INCLUDE [not-latest-version](~/includes/not-latest-version.md)]
:::moniker-end
```

Reply:
Put that at the top of the file like I said when I mentioned you in another PR.

Reply:
It does [this](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/host/hosted-services?branch=pr-en-us-28734&view=aspnetcore-5.0&tabs=visual-studio)

Reply:
Thanks. Are we setting this include to show for version 7 only after 8 goes GA, or now during preview?

Reply:
If the latest released version is not selected, the warning is displayed.